src/main/java/com/smartling/marketo/sdk/rest/command/email/UpdateEmailFullContent.java

```diff
@@ -0,0 +1,45 @@
+package com.smartling.marketo.sdk.rest.command.email;
+
+import com.smartling.marketo.sdk.rest.Command;
+
+import java.lang.reflect.Type;
+import java.util.Collections;
+import java.util.Map;
+
+public class UpdateEmailFullContent implements Command<Void>
+{
+
+    private final int id;
+    private final String content;
+
+    public UpdateEmailFullContent(int id, String content)
+    {
+        this.id = id;
+        this.content = content;
+    }
+
+
+    @Override
+    public String getPath()
+    {
+        return "/asset/v1/email/" + id + "/fullContent.json";
+    }
+
+    @Override
+    public String getMethod()
+    {
+        return "MULTIPART";
```

Comment:
Do this work for you? As I understand this should be GET or POST

Reply:
No, I need to pass multipart file and sdk provides this possibility with "MULTIPART" version. Yes, it works)